src/Sylius/Bundle/CoreBundle/Resources/config/services.xml

```diff
@@ -586,6 +594,28 @@
             <argument type="service" id="doctrine" />
             <tag name="validator.constraint_validator" alias="sylius_has_enabled_entity" />
         </service>
+        <service id="sylius.controller.sitemap" class="%sylius.controller.sitemap.class%">
+            <argument type="service" id="sylius.sitemap_renderer.sitemap_renderer"/>
+            <argument type="service" id="sylius.sitemap_service.builder" />
+        </service>
+        <service id="sylius.sitemap_renderer.twig_adapter" class="%sylius.sitemap_renderer.twig_adapter.class%">
+            <argument type="service" id="templating" />
+            <argument>%sylius.sitemap%</argument>
+        </service>
+        <service id="sylius.sitemap_renderer.sitemap_renderer" class="%sylius.sitemap_renderer.sitemap_renderer.class%">
+            <argument type="service" id="sylius.sitemap_renderer.twig_adapter" />
+        </service>
+        <service id="sylius.sitemap_factory.sitemap" class="%sylius.sitemap_factory.sitemap.class%" />
+        <service id="sylius.sitemap_factory.sitemap_url" class="%sylius.sitemap_factory.sitemap_url.class%" />
+        <service id="sylius.sitemap_service.builder" class="%sylius.sitemap_service.sitemap_builder.class%">
+            <argument type="service" id="sylius.sitemap_factory.sitemap" />
+        </service>
+        <service id="sylius.sitemap_provider.product" class="%sylius.sitemap_provider.product_url.class%" >
```

Comment:
@lchrusciel You will group all kinds of providers under ``sylius.product.X`` key? This does not make sense, we should be specific what kind of provider it is, so current name is better.